packages/console/src/monaco/MonacoProviders.tsx

```diff
@@ -251,15 +251,17 @@ class MonacoProviders extends PureComponent<
       };
     }
 
-    const diagnostics = MonacoProviders.getDiagnostics(model).filter(d => {
-      const diagnosticRange = new monaco.Range(
-        d.location.row,
-        d.location.column,
-        d.end_location.row,
-        d.end_location.column
-      );
-      return diagnosticRange.intersectRanges(range);
-    });
+    const diagnostics = MonacoProviders.getDiagnostics(model)
+      .filter(d => d.code != null) // Syntax errors have no code and can't be fixed/disabled
```

Comment:
small nitpick. I would probably combine these into 1 callback to avoid 2 passes of the array

Reply:
Good point